lib/src/st2110/pipeline/st20_pipeline_rx.c

```diff
@@ -217,6 +217,8 @@ static int rx_st20p_frame_ready(void* priv, void* frame,
   framebuff->src.timestamp = framebuff->dst.timestamp = meta->timestamp;
   framebuff->src.rtp_timestamp = framebuff->dst.rtp_timestamp = meta->rtp_timestamp;
   framebuff->src.status = framebuff->dst.status = meta->status;
+  framebuff->src.receive_timestamp = framebuff->dst.receive_timestamp =
+      meta->timestamp_first_pkt;
```

Comment:
same as above please include the same for st30 and st40, 
may not be used in the end but still